src/main/java/frc/robot/Constants.java

```diff
@@ -116,9 +116,9 @@ public static final class DriveConstants {
 	}
 
 	public static final class FlywheelConstants {
-		public static final int kMasterPort = 80; // 9 valk
-		public static final int kFollowerPort = 81; // 6 valk
-		public static final boolean kMasterInvert = true; // TODO: All below were from last year, change
+		public static final int kMasterPort = 49;
```

Comment:
Unless you have heard differently the bottom flywheel is apparently the master so 50 and 49 for the other.